nextflow_schema.json

```diff
@@ -144,7 +144,7 @@
         },
         "krona_taxonomy_tab": {
             "type": "string",
-            "default": "assets/databases/krona/taxonomy/taxonomy.tab"
+            "default": "/home/proj/development/microbial/metagenomics/sofia_workspace/gms_16S/assets/databases/krona/taxonomy/taxonomy.tab"
```

Comment:
Please remove user-specific part in the path.

Reply:
This was by mistaked when I was tested, I will remove this.